actors/abi/epoch.go

```diff
@@ -0,0 +1,8 @@
+package abi
+
+// The duration of a chain epoch.
+// This is used for deriving epoch-denominated periods that are more naturally expressed in clock time.
+const EpochDurationSeconds = 25
+const SecondsInYear = 31556925
+const SecondsInDay = 86400
+const SecondsInHour = 3600
```

Comment:
I think all of the uses of these are for epochs, so precompute EpochsInDay, etc, here.